Lib/importlib/_bootstrap_external.py

```diff
@@ -280,7 +282,7 @@ def _write_atomic(path, data, mode=0o666):
 # Whenever MAGIC_NUMBER is changed, the ranges in the magic_values array
 # in PC/launcher.c must also be updated.
 
-MAGIC_NUMBER = (3420).to_bytes(2, 'little') + b'\r\n'
+MAGIC_NUMBER = (3421).to_bytes(2, 'little') + b'\r\n'
```

Comment:
No magic number update is needed. The bytecode itself will not be changed, just human names of some opcodes.